pysumma/ensemble.py

```diff
@@ -154,9 +156,11 @@ def start(self, run_option: str='local', prerun_cmds: list=None):
             # Sleep calls are to ensure writeout happens
             config = self.configuration[n]
             self.submissions.append(self._client.submit(
-                _submit, s, n, run_option, prerun_cmds, config))
+                _submit, s, n, run_option, run_suffix, prerun_cmds, freq_restart,write_config,write_manager,config))
```

Comment:
Could you split this so that it doesn't end up being a long line?

Reply:
Done